README.md

```diff
@@ -198,42 +235,46 @@ Breaking down this command:
 2. **-o**: This represents options. 
 In our code, we have a dictionary of options to choose from, like `sides` and `page-ranges`.
 3. **-d**: This represents the destination printer.
-4. **JerryLeeResume.pdf**: This is my resume, 
-and if you are a recuiter, please download and read it.
-This was used as test pdf to print.
+4. **sample.pdf**: This is the PDF file that will be sent to the printer.
 
 To understand more about LPD, type in `man lp` in terminal or visit
 the [LP Manual Page](https://www.cups.org/doc/man-lp.html).
 
-## Difficulties Encountered
+### Difficulties Encountered
 
-### Using IPP Protocol
+#### Using IPP Protocol
 Initially, we tried using the IPP (Internet Printing Protocol) protocol to 
 print. However, the printer was too old so it was unable to 
 use the protocol. When we tried printing a file using IPP, the 
 unformatted raw pdf of the file was printed.
 
-### Finding the Correct Protocol
+#### Finding the Correct Protocol
 We soon realized that the printers were manufactured in 2006, and still used 
 LPD (Line Printer Daemon Protocol).
 
-### Establishing a Connection to the Printer
+#### Establishing a Connection to the Printer
 We were unable to connect to the printers through the school's wifi. We later 
 saw that the printers had an ethernet cable connected to them. We found the IP 
 address by printing out the network configuration page and established a 
 successful connection through ethernet.
 
-### Referencing the Existing system
+#### Referencing the Existing system
 We attempted to look at the old working printing page for
 clues. However, the old code consisted of tens, possibly 
 hundreds, of thousands of lines that were undocumented. 
 It was difficult to follow the code since it was all over
 the place and repetitive. The repository of the existing system can be found 
 [here](https://github.com/SCE-Development/Printer/)
 
-## Helpful links
+### Helpful links
 [gRPC in Python](https://www.semantics3.com/blog/a-simplified-guide-to-grpc-in-python-6c4e25f0c506/)
 
 [HP p2015dn Manual](http://www.hp.com/ctg/Manual/c00623611.pdf)
 
 [LP Command Manual Page](https://www.cups.org/doc/man-lp.html)
+
+
+## LED sign
```

Comment:
we can keep this for now, but i am thinking we move all documentation to the [Core-v4 Wiki](https://github.com/SCE-Development/Core-v4/wiki) to avoid a super long README